packages/vector/src/services/chroma-provider.ts

```
@@ -0,0 +1,123 @@
+    } catch (error) {
+      throw new Error(`Failed to connect to ChromaDB: ${error}`)
+    }
+  }
+  
+  async createCollection(name: string, dimension?: number): Promise<void> {
+    try {
+      await this.client.createCollection({
+        name,
+        embeddingFunction: this.embedder,
+      })
+    } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)